lib/theme.json

```diff
@@ -236,5 +236,8 @@
 				}
 			}
 		}
+	},
+	"styles": {
+		"spacing": { "blockGap": "24px" }
```

Comment:
I'm noticing an issue if I set a gap for 60px. And I add two top-level paragraphs in a post. On the front end, the gap between paragraphs is 60px, on the template editor too but on the post editor, the gap is not the 60px.

Reply:
Which theme are you using, I can take a look.

Reply:
The tt1-blocks from theme-experiments repo.

Reply:
For me it works properly 🤔